### ISSUE TITLE
Modify the column names for the existing SEEPS line type in the database schema

### ISSUE THREAD

**Issue by JohnHalleyGotway** [dtcenter/METdatadb#335]
## Describe the Enhancement ##
This issue is the downstream impact of dtcenter/MET#2882. For this MET issue, @mpm-meto and @RachelNorth are changing the names of the existing SEEPS columns to provide more clarity and make them more intuitive:
`S12` -> `ODFL` for the observation dry, forecast light category, 
`S13` -> `ODFH` for the observation dry, forecast heavy category,
`S21` -> `OLFD` for the observation light, forecast dry category,
`S23` -> `OLFH` for the observation light, forecast heavy category,
`S31` -> `OHFD` for the observation heavy, forecast dry category,
`S32` -> `OHFL` for the observation heavy, forecast light category

Recommend making this change only after dtcenter/MET#2882 is complete and a corresponding PR merges these changes into the develop branch.

Note that @mpm-meto has indicated that backward-compatibility is NOT required. If changes for this this issue no longer support loading v11.1 SEEPS output into METdataio, that's OK. If backward compatibility is easily added, please do so. If it's not easy, don't worry about it.

@bikegeek please also consider whether changes are needed in other METplus Analysis Tools repositories.

### Time Estimate ###
2-3 days @bikegeek please update

### Sub-Issues ###
Consider breaking the enhancement down into sub-issues.
None needed

### Relevant Deadlines ###
While the MetOffice funding will be extended until March 2025, it would be ideal to include the SEEPS changes in the METplus 6.0.0 beta6 development release for the MetOffice to test and validate.

### Funding Source ###
MetOffice: 2799991 (coordinate with @j-opatz)

### Assignee ###
- [x] Select **engineer(s)** or **no engineer** required
- [x] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] Review default **alert** labels
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Milestone and Projects ###
- [x] Select **Milestone** as a **METdataio-X.Y.Z** version, **Consider for Next Release**, or **Backlog of Development Ideas**
- [x] For a **METdataio-X.Y.Z** version, select the **METdataio-X.Y.Z Development** project

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdataio](https://github.com/dtcenter/METdataio/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Enhancement Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/appendixA.html#metplus-components-python-packages) table.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issue
Select: **Milestone** as the next official version
Select: **METdataio-X.Y.Z Development** project for development toward the next official release
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

**Comment by jprestop** [comment]
Closed with PR #340
